src/control/server/mgmt_pool.go

```diff
@@ -699,7 +699,7 @@ func (svc *mgmtSvc) PoolDestroy(parent context.Context, req *mgmtpb.PoolDestroyR
 
 		// Perform separate PoolEvict _before_ possible transition to destroying state.
 		evStatus, err := svc.poolEvictConnections(ctx, req)
-		if err != nil {
+		if !req.Force && (err != nil) {
 			return nil, err
```

Comment:
In force mode, do we want to log the errors whilst ignoring them?

Reply:
I'm looking at changing the Debugf to Errorf within poolEvictConnections() for this